docs/user-guide/ui.md

```diff
@@ -32,19 +32,21 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
+<!-- TODO(bryk): Update the documentation. -->
```

Comment:
I was going to do this in a few small pull requests. But OK, I'll do this in one PR.